mmdet/models/dense_heads/yolo_head.py

```diff
@@ -0,0 +1,595 @@
+# Copyright (c) 2019 Western Digital Corporation or its affiliates.
+
+import logging
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule, xavier_init
+from mmcv.runner import load_checkpoint
+
+from mmdet.core import force_fp32, multiclass_nms
+from ..builder import HEADS
+from .base_dense_head import BaseDenseHead
+
+_EPSILON = 1e-6
+
+
+@HEADS.register_module()
+class YOLOV3Head(BaseDenseHead):
+    """
+    YOLOV3Head
+    Add a few more conv layers and generate the output.
```

Comment:
We may replace this line with paper link.

Reply:
Sure.